.appveyor.yml

```diff
@@ -24,8 +24,8 @@ environment:
     # Testing other channels
     - TARGET: x86_64-pc-windows-gnu
       RUST_VERSION: nightly
-    #- TARGET: x86_64-pc-windows-msvc
-    #  RUST_VERSION: nightly
+    - TARGET: x86_64-pc-windows-msvc
```

Comment:
We were not testing this before. How much time does this add to the build? Is it really worth it at this stage of the project?

Reply:
A lot - not because of the builds but beause appveyor takes a lot of time to schedule the jobs. At the time I set this up MSVC was not recommended, but apparently now it is well supported (or more so than the gnu variant), so we might as well switch to it. Additionally, this build is not required and we should not be waiting for it in any case, but it's good to know if/when it breaks.

Reply:
I'll let you decide whether we should use msvc or gnu.
The PR looks good to me :+1: